sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSender.java

```diff
@@ -271,6 +272,17 @@ public Mono<Void> send(List<Message> messageBatch, DeliveryState deliveryState)
                 final Message batchMessage = Proton.message();
                 batchMessage.setMessageAnnotations(firstMessage.getMessageAnnotations());
 
+                // Set partition identifier properties of the first message on batch message
+                if ((firstMessage.getMessageId() instanceof String)
+                    && !CoreUtils.isNullOrEmpty((String) firstMessage.getMessageId())) {
+
+                    batchMessage.setMessageId(firstMessage.getMessageId());
```

Comment:
nit; extra new line

Reply:
Ahhh, that was intentional because I find it easier to understand what is going on inside the blob when the conditional statement spans multiple lines.